source/users/custom-metadata.txt

```diff
@@ -107,6 +107,67 @@ collection. Depending on your use case, you may:
   standard CRUD operations from a Function, an Atlas Device SDK, a MongoDB
   driver, or MongoDB Compass.
 
+Modify User Permissions in Custom User Data Documents
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:ref:`Permissions <permissions>` will automaticlly refresh for a user when you 
+change their custom user data document. Their :ref:`user session <user-sessions>` 
+will terminate and then reconnect automatically. 
+
+For user permissions to refresh automatically, the custom user data 
+documents should be stored in a normal collection and
+not in a :manual:`view </core/views/>` or :manual:`time series </core/timeseries-collections/>` collection. 
+
+For permissions to refresh automatically, don't delete a custom user data document. 
+Rather, unset all the non-ID fields in the document. 
+
+.. example::
+
+  Consider a collection named ``employees`` where each employee has their own 
+  document stating their level at the company. This collection might have 
+  three roles: Junior, Senior, or Manager. 
+  
+  The ``level`` field in the employee's custom user data document also has 
+  Junior, Senior, or Manager values. Using the ``apply_when`` expression, the 
+  role for the employee is based on the value of the ``level`` field. For example, 
+  the ``apply_when`` expression for the role of Senior would look like:
+
+  .. code-block:: json
+    :caption: Senior role with apply_when expression
+
+    {
+      "name": "Senior"
+      "apply_when": {"%%user.custom_data.level": "senior"},
+      ...
+    }
+
+  Say you you would like to remove the following employee's document
+  because they no longer work at the company:
+
+  .. code-block:: json
+    :caption: Custom user data document
+
+    {
+      "_id": "63ed2erealobjectid78e526",
+      "user_id": "63ed2dbe5960df2af7fd216e",
+      "level": "senior"
+    }
+  
+  First, you need to correctly remove the employee's permissions, by unsetting
+  the non-ID fields. The document would then look like:
+
+  .. code-block:: json
+    :caption: Correctly updated custom user data document
+
+    {
+      "_id": "63ed2erealobjectid78e526",
+      "user_id": "63ed2dbe5960df2af7fd216e"
+    } 
+
+  Unsetting the non-ID field allows App Services to remove the employee's 
```

Comment:
There's a slight detail where if you delete the document instead of unsetting the field, we just won't _automatically_ update the employee's permissions. The permissions would still be updated if the end user closes and re-opens their app, for example.

Also, deleting the custom user data document doesn't necessarily mean that the employee's permissions are removed, as they could still have some permissions if the developer has a role set up with an empty `apply_when` expression as a catch-all.